### DISCUSSION TITLE
How semantic release is managed for this repo

### DISCUSSION THREAD

**Discussion by ZhengRui** [pmndrs/zustand#1147] [General] [answered]
I am looking into improving the semantic versioning and releasing of my own projects, and I really like how this repo is organized and I want to learn how commits/tags/releases are managed for this repo. I searched and found some tools like `semantic-release/semantic-release`, `commitzen/cz-cli`, but I couldn't find any trace of these tools being used in this repo, so I am wondering what are the tools and practices you are using for managing commits/tags/releases ? Thanks.

**Comment by dai-shi** [comment, marked as answer]
We don't use any tools at this moment.
